x/oracle/client/cli/tx.go

```diff
@@ -341,3 +345,131 @@ func GetCmdDeleteTask() *cobra.Command {
 
 	return cmd
 }
+
+func GetCmdRespondToTxTask() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "respond-to-txtask <tx_hash> <score>",
+		Short: "Respond to a transaction task",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			txf := tx.NewFactoryCLI(cliCtx, cmd.Flags()).WithTxConfig(cliCtx.TxConfig).WithAccountRetriever(cliCtx.AccountRetriever)
+
+			from := cliCtx.GetFromAddress()
+			if err := txf.AccountRetriever().EnsureExists(cliCtx, from); err != nil {
+				return err
+			}
+
+			txHash, err := base64.StdEncoding.DecodeString(args[0])
+			if err != nil {
+				panic(err)
+			}
+
+			score, err := strconv.ParseInt(args[1], 10, 64)
+			if err != nil {
+				panic(err)
+			}
+
+			msg := types.NewMsgTxTaskResponse(txHash, score, from)
+			if err := msg.ValidateBasic(); err != nil {
```

Comment:
tx.GenerateOrBroadcastTxWithFactory will do msg.ValidateBasic.

Reply:
it makes sense. 
done.